app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,13 @@
+class FriendshipsController < ApplicationController
+
+  def destroy
+    if user = User.find_by(id: params[:user_id])
```

Comment:
I'm not sure this is right. We are using the `if` statement but we are assigning `user` to `User.find_by(....)`. We need to fix this.

Another thing, once we fix #3 we perhaps won't need the `destroy_friendship` method in the `User` model anymore.